src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
```

Comment:
Going by pure boolean logic, the reverse of `isNotEditable` would be:

```
boolean isEditable = editInstructorNameTextBox.isEnabled()
                     || editInstructorEmailTextBox.isEnabled()
                     || saveButton.isEnabled();
```

which probably doesn't make too much sense either.

And yep renaming it to `isInstructorEditFormDisabled` would be good :)